web/.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 build
 dist
-package-lock.json
```

Comment:
We want this so the CI doesn't have to recalculate the dependency tree on every run.
Also means deterministic builds in a pipeline vs. locally when it comes to that